app/models/generic_object.rb

```diff
@@ -14,13 +14,11 @@ class GenericObject < ApplicationRecord
     :string   => ActiveRecord::Type::String.new,
   }.freeze
 
-  def must_be_defined_attributes
-    return errors.add(:base, "must specify a GenericObjectDefinition.") unless generic_object_definition
-
-    found = custom_attributes.detect { |ca| !generic_object_definition.defined_attributes.include?(ca.name) }
-    if found
-      errors.add(:base, "#{found.name} is not defined in GenericObjectDefinition: #{generic_object_definition.defined_attributes.keys.join(", ")}.")
-    end
+  def initialize(attributes = {})
+    attributes = (attributes || {}).stringify_keys
```

Comment:
Most callers will call with symbols, so I think prefer `symbolize_keys`.  That way, the operation will be a no-op in the majority of cases.